oaknational/Oak-Web-Application
# Fix/migration copyright bugs mv update - LESQ-658

## Description

Music year: {owa_music_year}

- Fixes some of the bugs for the migrated EYFS lessons

1. Should be no more 404 share pages
2. Lesson overview download buttons should be there when expected
3. Copyright download pages do not have downloadable resources


## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2278--oak-web-application.netlify.thenational.academy
Everything should be the same as it was before, as no published EYFS.

## Screenshots - from locally published EYFS data

**Top download all button**

How it used to look (delete if n/a):
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/bee16260-d6e0-4fbd-b7db-fa4b352f0edc)

How it should now look:
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/368e1f20-cb22-4cd8-8e69-0398ec8ce858)

**Share 404 pages**

How it used to look (delete if n/a):
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/cc514d52-ad18-4c4c-8ff8-911590dfc7f4)

How it should now look:
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/4d9bd22c-010b-4b35-b5ad-5c2f21102db2)

**Copyrighted lesson download page** (this should not be linked to but will still exist)

How it used to look (delete if n/a):
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/b2dd810f-4b20-4cb1-bd2e-92bf40bfb6fb)


How it should now look:
![image](https://github.com/oaknational/Oak-Web-Application/assets/7987787/bd6ef434-43a8-4191-b959-4131fd9ad6d6)



## Checklist
